data/tlg0065/tlg001/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0065" xml:lang="grc" urn="urn:cts:greekLit:tlg0065.tlg001">
+  <ti:title xml:lang="lat">Periplus Scylacis</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg0065.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0065.tlg001">
+    <ti:label xml:lang="lat">Periplus Scylacis, Geographi graeci minores Volumen Primum</ti:label>
+    <ti:description xml:lang="eng">Scylax, of Carianda, active 6th century B.C, creator; Müller, Karl, of Paris, editor</ti:description>
```

Comment:
As with the other work-level `__cts__.xml` file, add a bit more bibliographical information here (publisher and year of publication).